notebooks/source/model_rendering.ipynb

```diff
@@ -0,0 +1,149 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "happy-temperature",
+   "metadata": {},
+   "source": [
+    "# Automatic rendering of NumPyro models\n",
+    "\n",
+    "In this tutorial we will demonstrate how to create beautiful visualizations of your probabilistic graphical models."
```

Comment:
Having a notebook is a great idea. I'll craft several examples which you can adapt for the tutorial. :)

Reply:
Awesome :-)

Reply:
This might be beyond the purpose of this PR: I think you can port some plots in [this gist](https://nbviewer.jupyter.org/gist/fehiepsi/1b2abcb8f073d0a555aa9264e3dcc8ef) to this tutorial. It would be nice to say a bit about `unflatten` feature, `render_distributions` feature, and the difference in using `reparam` in `logistic_random_effects` model.

Reply:
I'd be happy to do so!
Do you think the `unflatten` feature is useful enough to include its usage with an extra argument to `render_model`?

Reply:
My impression so far is it is tricky to choose a good `unflatten` value. It might be better to let users decide what they want. We might introduce it in the tutorial. It is also helpful to show users how to save the graph to a file there (probably a link to the graphviz API docs for saving is enough) - personally, I would like to play with the graph and save it in various formats without having to rerun `render_model`. :)

Reply:
Agreed, that makes sense!